tdd_rdf_ogm.py

```
@@ -0,0 +1,199 @@
+xyz -> lit(xyz)
+abc\:xyz -> lit(abc:xyz)
+
+None -> bnode(_:uuid) IF kind=blank
+None -> uri(:uuid) IF kind=iri
+None -> uri(uuid) IF kind=relative
+
+inode -> identifier is an IRI
+rnode -> identifier is a relative IRI
+bnode -> identifier is a blank node
+"""
+from rdflib import Graph, Literal, Namespace, URIRef
```

Ruff (0.12.2)

`rdflib.Graph` imported but unused

Remove unused import

(F401)